org.duckstation.DuckStation.yml

```diff
@@ -0,0 +1,52 @@
+app-id: org.duckstation.DuckStation
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: duckstation-nogui
+finish-args:
+  - --device=all
+  - --filesystem=host:ro
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+
+modules:
+  - name: libevdev
+    buildsystem: meson
+    config-opts:
+      - -Dtests=disabled
+      - -Ddocumentation=disabled
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /share
+    sources:
+      - type: archive
+        url: https://www.freedesktop.org/software/libevdev/libevdev-1.11.0.tar.xz
+        sha256: 63f4ea1489858a109080e0b40bd43e4e0903a1e12ea888d581db8c495747c2d0
+
+  - name: duckstation
+    buildsystem: simple
+    builddir: true
+    build-commands:
+      - cmake -G Ninja -DCMAKE_BUILD_TYPE=Release -DUSE_SDL2=ON -DUSE_WAYLAND=1 -DUSE_DRMKMS=1
```

Comment:
You could specify `no-make-install: true` since you say the install target is missin, and use `cmake-ninja` buildsystem. And put these install steps into the `post-install`